Update whats-new-in-defender-for-office-365.md (#5652)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md

> [!TIP]
> Don't have Microsoft Defender for Office 365 yet? [Contact sales to start a trial](https://info.microsoft.com/ww-landing-M365SMB-web-contact.html).

## August 2021

> - [Admin review for reported messages](admin-review-reported-message.md): admins can now send templated messages back to end users after they review reported messages. This can be customized for your organization and based on your admin's verdict as well. 

Reviewer: ```suggestion
- [Admin review for reported messages](admin-review-reported-message.md): After admins have reviewed reported messages, they can now send templated messages back to users. These can be customized for your organization and can also be based on your admin's verdict. 
```

> - [Admin review for reported messages](admin-review-reported-message.md): admins can now send templated messages back to end users after they review reported messages. This can be customized for your organization and based on your admin's verdict as well. 

Reviewer: ```suggestion
- [Admin review for reported messages](admin-review-reported-message.md): Admins can now send templated messages back to end users after they review reported messages. This can be customized for your organization and based on your admin's verdict as well. 
```
